portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,42 +40,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  static String DATA_TYPE = "Comment";
-  static String COMMENT_TIMESTAMP = "timestamp";
-  static String COMMENT_AUTHOR = "name";
-  static String COMMENT_CONTENT = "message";
   static String MAX_COMMENTS_PARAM = "max";
   static String PAGE_NUM_PARAM = "page";
-  static int PAGE_SIZE;
-  static int PAGE_NUM;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    PAGE_SIZE = getNumComments(request);
-    PAGE_NUM = Integer.parseInt(request.getParameter(PAGE_NUM_PARAM));
+    int PAGE_SIZE = getNumComments(request);
```

Comment:
Style nitpick: reserve ALL_CAPS vars for constants. Use camelCase for regular vars. I.e. `int pageSize` and `int pageNum`.